cartography/rules/formatters.py

```diff
@@ -0,0 +1,42 @@
+"""
+Output formatting utilities for Cartography rules.
+"""
+
+import re
+from urllib.parse import quote
+
+
+def _generate_neo4j_browser_url(neo4j_uri: str, cypher_query: str) -> str:
```

Comment:
There is also an ssc scheme for bolt and neo4j: https://neo4j.com/docs/python-manual/current/connect-advanced/